advanced/Scripts/webpage.sh

```diff
@@ -629,6 +629,7 @@ checkDomain()
 
 escapeDots()
 {
+    # shellcheck disable=SC2001
```

Comment:
This is to disable a false positive? If so, could you add a comment that says why the check was disabled?

Reply:
See the title of the second commit. SH suggests `${variable//search/replace}` which is bash-only.

Reply:
I forced push with a more detailed explanation.

Reply:
I don't think this is a false positive. `<<<` is bash only.

I think the POSIX alternative is:

```sh
    escaped=$(echo "$1" | sed 's/\./\\./g')
```